config/locales/views/en.yml

```
@@ -1599,6 +1599,17 @@ en:
       name: Name
       submit: Submit
       ticket_footnote: Numbers only.
+  readings:
+    index:
+      page_heading: History
+      content_heading: List of History Items
+      full_history: Full History
+      marked_for_later: Marked for Later
+      clear_entire_history: Clear Entire History
+    confirm_clear:
+      page_heading: Clear Entire History
+      notice: Are you sure you want to <strong><em>clear</em></strong> your entire history? This <strong>cannot be undone</strong>.
```

This needs to be adjusted as well, sorry for not noting that in my previous review!
```suggestion
      notice_html: Are you sure you want to <strong><em>clear</em></strong> your entire history? This <strong>cannot be undone</strong>.
```